static/js/overlays.js

```diff
@@ -201,8 +201,6 @@ exports.close_modal = function (selector) {
 
     const elem = $(selector).expectOne();
     elem.modal("hide").attr("aria-hidden", true);
-    // Enable mouse events for the background as the modal closes.
-    exports.enable_background_mouse_events();
```

Comment:
This is causing models that we have in settings/manage organization like "change password" or "change name" etc to not re-enable mouse events even after the modal is closed. When the puppeteer test changes the password(then waits for the modal to be closed) and then clicks on "alert words" section but the I think the click isn't working because of this change.

(Also tested your PR manually to verify that clicks are actually not re-enabled)